conda_env/cli/main_vars.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from argparse import RawDescriptionHelpFormatter
+
+from conda.cli import common
+from conda.cli.conda_argparse import add_parser_prefix, add_parser_json
+from conda.core.envs_manager import get_environment_env_vars, set_environment_env_vars, \
+    unset_environment_env_vars
+from conda.base.context import context
+from .common import get_prefix
+
+
+description = """
+Interact with environment varaibles associated with Conda environments
+"""
+
+example = """
+examples:
+    conda env vars --list -n my_env
+    conda env vars --set MY_VAR=something
+    conda env vars --unset MY_VAR
+"""
```

Comment:
This is looking pretty close to the configuration specific to an environment. Since we're talking about moving channel configuration and a handful of other config parameters to stick to a specific environment, in designing this CLI we should also take into account what the general case will look like.

Seems like `conda env config` is probably the start of the more general case for environment configuration.

Another thing to consider here is that `--flags` are often considered an anti-pattern for what you really intend to be commands.  That is, flags are meant to be optional modifications; if you write out the full list of command examples and every one requires at least one flag, then you've tripped on the anti-pattern.  The current conda CLI does this all over the place, but we need not repeat here as we're introducing new features.

Reply:
sounds good, happy to put this under `conda env config vars`. 
Regarding the flags, conda config currently uses flags for setting/appending/etc. Though this is an antipattern I think another thing that is important to consider is the patterns that users are already familiar with. If it is important that `conda env config ...` matches `conda config` then I think the flags should stay otherwise I'm more than happy to make them sub commands. What do you think?